### PR TITLE
feat(lib2): Setup code-base for lib 2

### LINKED ISSUE #9: Setup Code Base for Libary 2
## User story
1. As a team
2. I want to set up the development project
3. So that we have a good basis for software development

## Acceptance criteria
* project can run locally
* the code base is initalized
* the software project builds
* add readme how to build it locally

### PR DESCRIPTION
**Pull request by a-miscellaneous**
- added a readme to explain the library
- added an abstract class to assist in implementation

closes #9

### CODE REVIEW

File: DifferenceGenerator/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+### Installation
+
+1. Clone the project repository to your local machine
+2. Change your working directory to the project folder
+3. Build the project using `gradle build`
+4. Run the project using `gradle run`
+
+### Initialization
+
+Your video generator should extend the `AbstractVideoGenerator` class. You can then initialize your video generator.
```
Comment (simonsasse) [inline DifferenceGenerator/README.md:27]:
```suggestion
Your video generator should extend the `AbstractDifferenceGenerator` class. You can then initialize your difference generator.
```

**Review by simonsasse — CHANGES_REQUESTED** [review]
Nice, just a minor CnP error 💅

File: DifferenceGenerator/README.md (hunk 1 above)
Comment (a-miscellaneous) [reply DifferenceGenerator/README.md:27]: sure thing

**Comment by a-miscellaneous** [comment]
don't merge, still awaiting ktlint

**Comment by simonsasse** [comment]
#11 is merged, you can run `ktlint` and see what it has to say. I think all the things are should be fixed by running `ktlint -F`.  You should probably also run a `git rebase origin main`.

**Pull request merged**